scrapy/loader/__init__.py

```diff
@@ -174,11 +175,12 @@ def _process_input_value(self, field_name, value):
                                     value, type(e).__name__, str(e)))
 
     def _get_item_field_attr(self, field_name, key, default=None):
-        if isinstance(self.item, Item):
-            value = self.item.fields[field_name].get(key, default)
+        try:
+            field_meta = ItemAdapter(self.item).get_field_meta(field_name)
+        except TypeError:
```

Comment:
this exception can't be reaised now

Reply:
Oh nice catch, I only remembered the one in `exporters.py`